cmd/verify.go

```diff
@@ -83,16 +95,9 @@ func runVerify(ctx context.Context, vo options.VerifyOptions, verifiers ...crypt
 		verifiers = append(verifiers, v)
 	}
 
-	inFile, err := os.Open(vo.PolicyFilePath)
+	policyEnvelope, err := pkg.LoadPolicy(ctx, vo.PolicyFilePath, pkg.NewArchivistaClient(vo.ArchivistaOptions.Url, archivistaClient))
```

Comment:
So I reckon that we could benefit from splitting the options up between `--policy-file` and `--policy-sha`. We have gone down the route of a more declarative experience on Witness rather than the "magic" approach.

Having said this I am not 100% sure what the best flag definitions would be for this.

Reply:
I avoided it because we break compatibility with existent deployments, as it will break the verification.

I'm ok with doing a more declarative.
Should I go for this change?

Reply:
This is a good point that I hadn't fully considered. Let me think about it for a couple of minutes 😄